tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -904,4 +944,47 @@ public void exec(int depth, int concurrencyDepth, AlluxioURI path) throws Except
       }
     }
   }
+
+  class ConcurrentCreateDelete implements Callable<Void> {
```

Comment:
I made it `private` for now, but it accesses a member of the outer class, like the other Callables here. I say we have a later PR to change them to take in a parameter, and change to `static`.